### DISCUSSION TITLE
Support for arm64 docker images

### DISCUSSION THREAD

**Discussion by benfortuna** [websight-io/starter#108] [Q&A] [answered]
Hi, just wondering if you provide docker images for arm64 architecture, or plan to in the future?

I noticed AWS are offering a t4g.small instance on free tier for 12 months, so I would like to try websight nginx on this architecture.

Thanks.

**Comment by paweljozwicki** [comment, marked as answer]
Hi, 
our images ([cms](https://gallery.ecr.aws/ds/websight-cms-starter), [nginx](https://gallery.ecr.aws/ds/websight-nginx-starter)) are provided for ARM 64 and AMD 64 so you should be able to try them on t4g.small.
